testbook/client.py

```diff
@@ -96,6 +101,10 @@ def inject(self, func, args=None):
         else:
             raise TypeError('can only inject function or code block as str')
 
+        # Execute the pre-run cells if passed
+        if prerun or prerun == 0:
+            self.execute_cell(prerun)
```

Comment:
Should we support ranges / lists here of cells?

Reply:
We do support lists of cells. `execute_cell` can take in a list of cells or just a single cell (either as index or tag) and it will execute it.